dev-bin/make-man-pages.pl

```diff
@@ -67,18 +66,20 @@ sub _make_man {
             '-M', "section:$section",
             $input,
             '-o', $output,
-        );
+        ) == 0 or die "Failed to run lowdown: $!";
     }
 }
 
 sub _make_lib_man_links {
     my $target = shift;
 
-    my $header = read_file("$Bin/../include/maxminddb.h");
+    my $header = read_file("$Bin/../include/maxminddb.h")
+        or die "Failed to read header file: $!";
```

Comment:
It looks to me like `read_file` might already handle errors for us.

Reply:
Ah, yeah, at the very least autodie didn't cover them. Sorry about that.